examples/config.hocon.sample

```diff
@@ -220,35 +220,37 @@ collector {
 
     # Enable the chosen sink by uncommenting the appropriate configuration
     sink {
-      # Choose between kinesis, google-pub-sub, kafka, nsq, or stdout.
+      # Choose between kinesis, sqs, google-pub-sub, kafka, nsq, or stdout.
       # To use stdout, comment or remove everything in the "collector.streams.sink" section except
       # "enabled" which should be set to "stdout".
-      enabled = kinesis
+      enabled = kinesis # or sqs
       enabled = ${?COLLECTOR_STREAMS_SINK_ENABLED}
 
       # Region where the streams are located
-      region = {{kinesisRegion}}
+      region = {{kinesisRegion}} # or {{sqsRegion}}
```

Comment:
This setting is only for Kinesis and SQS. I agree that `sinkRegion` is more concise but won't there be a risk that it may confuse people who are trying to setup the pubsub collector for example?